querydsl-spatial/src/main/java/com/querydsl/spatial/GeometryPaths.java

```diff
@@ -25,6 +25,8 @@
 import org.geolatte.geom.Polygon;
 
 /**
+ * {@code GeometryPaths} provides factory methods for {@link GeometryExpression} creation
```

Comment:
The return types extend GeometryExpression & Path.
On 23 Apr 2015 23:24, "Ruben Dijkstra" <notifications@github.com> wrote:

> In querydsl-spatial/src/main/java/com/querydsl/spatial/GeometryPaths.java
> <https://github.com/querydsl/querydsl/pull/1325#discussion_r28999601>:
>
> > @@ -25,6 +25,8 @@
> >  import org.geolatte.geom.Polygon;
> >
> >  /**
> > + * {@code GeometryPaths} provides factory methods for {@link GeometryExpression} creation
>
> Shouldn't {@link GeometryExpression} be {@link GeometryPath}?
> I only see Path subtypes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/querydsl/querydsl/pull/1325/files#r28999601>.
>